### PR TITLE
Move is_deductible as it has nothing to do with how Sales Tax is handled

### PR DESCRIPTION
**Pull request by KarinG**
is_deductible on financial types is a yes/no for Income Tax purposes (generation of charitable receipts).

### CODE REVIEW

File: docs/financial/financialentities.md [hunk 1]
```diff
@@ -18,12 +18,15 @@ are not used functionally in CiviCRM but are available in various reports.
...
-In addition there are some functional fields that affect how tax is handled:
+In addition there are some functional fields that affect how Sales Taxes are calculated:
 
 * tax_rate
-* is_deductible
 * is_tax
 
+Use this field to indiciate whether a financial type is tax deductible for Income Tax purposes:
+* is_deductible 
+[note: CiviCRM itself does not calculate Charitable Tax Receipts - but extension like https://civicrm.org/extensions/cdn-tax-receipts do]
```
Comment (MikeyMJCO) [inline docs/financial/financialentities.md:28]:
```suggestion
!!! note
CiviCRM itself does not calculate Charitable Tax Receipts - but extensions like [CDN Tax Receipts](https://civicrm.org/extensions/cdn-tax-receipts) do.
```

File: docs/financial/financialentities.md [hunk 2]
```diff
@@ -18,12 +18,15 @@ are not used functionally in CiviCRM but are available in various reports.
 * contact id (not used in core but can be used in extensions - e.g. this is used in the [connectors](https://github.com/eileenmcnaughton/nz.co.fuzion.connectors) extension to link some accounts to one Xero account & others to a different one).
 * is_header_account
 
-In addition there are some functional fields that affect how tax is handled:
+In addition there are some functional fields that affect how Sales Taxes are calculated:
```
Comment (MikeyMJCO) [inline docs/financial/financialentities.md:21]:
```suggestion
In addition there are some functional fields that affect how sales taxes are calculated:
```

**Comment by MikeyMJCO** [comment]
A couple of small changes @KarinG then this is good to merge - great change to the clarity btw - thanks!

**Comment by KarinG** [comment]
Thanks! 👍

**Pull request merged**